from core.exceptions.base import CustomException

__all__ = ["CustomException"]
